maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -916,7 +958,7 @@ private boolean isValidIdWithWildCards( String id )
 
     private boolean isValidIdWithWildCardCharacter( char c )
```

Comment:
This methodname is now weird: we're going from an general `isValidIdWithWildCardCharacter` to a more explicit `isValidCoordinateIdCharacter`.

Reply:
ok, I'll rename this method also